elements/pfe-modal/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@patternfly/pfe-modal",
+  "pfelement": {
+    "className": "PfeModal",
+    "elementName": "pfe-modal"
+  },
+  "version": "1.0.0-prerelease.17",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Modal element for PatternFly Elements",
+  "keywords": [
+    "web-components",
+    "html"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "github:patternfly/patternfly-elements",
+    "directory": "elements/pfe-modal"
+  },
+  "main": "pfe-modal.umd.js",
+  "module": "pfe-modal.js",
+  "scripts": {
+    "build": "../../node_modules/.bin/gulp && ../../node_modules/.bin/prettier --ignore-path ../../.prettierignore --write '**/*.{js,json}'",
+    "dev": "../../node_modules/.bin/gulp dev",
+    "test": "../../node_modules/.bin/wct --configFile ../../wct.conf.json elements/pfe-modal/test/"
+  },
+  "author": {
+    "name": "cd"
+  },
+  "contributors": [
+    {
+      "name": "castastrophe",
+      "url": "https://github.com/castastrophe"
+    }
+  ],
+  "license": "MIT",
+  "dependencies": {
+    "@patternfly/pfe-sass": "1.0.0-prerelease.17",
```

Comment:
These will need to be bumped to prerelease.19

Reply:
Thanks! Made that fix.